app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -83,31 +83,21 @@ def start
   def pod_wait
     _log.info("waiting for pod #{pod_full_name} to be available")
 
-    client       = kubernetes_client
-    health_url   = pod_proxy_url(client, INSPECTOR_HEALTH_PATH)
-    http_options = {
-      :use_ssl     => health_url.scheme == 'https',
-      :verify_mode => ext_management_system.verify_ssl_mode,
-      :cert_store  => ext_management_system.ssl_cert_store,
-    }
-
-    # TODO: move this to a more appropriate place (lib)
-    response = pod_health_poll(client, health_url, http_options)
-
-    case response
-    when Net::HTTPOK
+    begin
+      ready = kubernetes_client.get_pod(options[:pod_name], options[:pod_namespace])[:status][:containerStatuses][0][:ready]
+    rescue SocketError, KubeException => e
+      msg = "unknown access error to pod #{pod_full_name}: #{e.message}"
+      _log.info(msg)
+      queue_signal(:abort_job, msg, "error")
```

Comment:
Won't this keep queuing pod_wait if there is an error? (line 99)
Maybe you need: 
```
return queue_signal(:abort_job, msg, "error")
```

Also do you mind adding brackets around e.message:
```
msg = "unknown access error to pod #{pod_full_name}: [#{e.message}]"
```
It makes nils really obvious